src/getMigrateDashboards.ts

```diff
@@ -32,7 +31,9 @@ export const getMigrateDashboards =
       doesReportIncludeStacks: boolean;
     },
   ) =>
-  (callback: MigrateDashboardsCallback): void => {
+  <FromDashboardState, ToDashboardState>(
+    callback: MigrateDashboardsCallback<FromDashboardState, ToDashboardState>,
```

Comment:
It is indeed a callback passed to the curried `migrateDashboards` function.
But its contract is to migrate a single dashboard, so I would call it `MigrateDashboardCallback`.
What do you think ?

Reply:
Yeah, I agree. The callback migrates only one dashboard, so it's type should reflect that, good point 👍